website/docs/guides/version-3-upgrade.html.md

```diff
@@ -615,6 +616,29 @@ resource "aws_autoscaling_group" "example"{
 }
 ```
 
+## Resource: aws_cloudfront_distribution
+
+### active_trusted_signers Attribute Name and Type Change
+
+Previously, the `active_trusted_signers` computed attribute was implemented with a Map that did not support accessing its computed `items` attribute in Terraform 0.12. 
```

Comment:
It could be accessed, just "unexpectedly weird" 😄 

```suggestion
Previously, the `active_trusted_signers` computed attribute was implemented with a Map that did not support accessing its computed `items` attribute in Terraform 0.12 correctly. 
```